<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件并运行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>基本功能</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>121</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="DCTButton">
      <property name="text">
       <string>DCT </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="DFTButton">
      <property name="text">
       <string>DFT </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="zhifangButton">
      <property name="text">
       <string>直方图均匀</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="grayButton">
      <property name="text">
       <string>灰度级</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="noseButton">
      <property name="text">
       <string>加噪</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>110</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="buguize">
      <property name="text">
       <string>buguize</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="kuang">
      <property name="text">
       <string>test_kuang</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fenge">
      <property name="text">
       <string>test_fenge</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="duishu">
      <property name="text">
       <string>duishu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="face">
      <property name="text">
       <string>test_face3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>78</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="junzhi">
      <property name="text">
       <string>junzhi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="gamma">
      <property name="text">
       <string>gamma</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
